src/main/Config.h

```diff
@@ -414,4 +414,7 @@ class Config : public std::enable_shared_from_this<Config>
     // function to stringify a quorum set
     std::string toString(SCPQuorumSet const& qset);
 };
+// A special name to be used for stdin in stead of a file name in command line
+// arguments.
+const std::string stdinSpecialName = "/dev/stdin";
```

Comment:
define this in the `Config` namespace - see `POSSIBLY_PREFERRED_EXTRA` ; also not sure, but you may run into a similar link error that I did in https://github.com/stellar/stellar-core/pull/2745 where I had to assign the value in the cpp file (link error only on some versions of clang it looks like)

Reply:
also, our naming convention for constants would make this look more like `STDIN_SPECIAL_NAME`

Reply:
Updated the constant name to `STDIN_SPECIAL_NAME` and moved it inside the `Config` namespace. Just to be safe, I also assigned the value in the cpp file.